docs/public/SUMMARY.md

```diff
@@ -0,0 +1,110 @@
+- [Installation]()
+    - [Prerequisites](installation/prerequisites.md)
+    - [Zonemaster-LDNS](installation/zonemaster-ldns.md)
+    - [Zonemaster-Engine](installation/zonemaster-engine.md)
+    - [Zonemaster-CLI](installation/zonemaster-cli.md)
+    - [Zonemaster-Backend](installation/zonemaster-backend.md)
+    - [Zonemaster-GUI](installation/zonemaster-gui.md)
+    - [Docker](installation/docker.md)
+- [Upgrade]()
+    - [Backend](upgrade/backend.md)
+- [Configuration]()
+    - [Profiles](configuration/profiles.md)
+    - [Backend](configuration/backend.md)
+        - [Environment Variables](configuration/backend-environment-variables.md)
+    - [GUI](configuration/gui.md)
+- [Using]()
+    - [CLI](using/cli.md)
+    - [RPCAPI](using/getting-started.md)
+        - [RPCAPI Reference](using/backend-api.md)
+    - [Telemetry](using/telemetry.md)
```

Comment:
And again, `backend-telemety.md`. 

Maybe it would make sense to put all that in a `backend` subdirectory / section? Or maybe this document could become a general telemetry document for both backend and engine? But that would require some rewording later.

Reply:
> Maybe it would make sense to put all that in a backend subdirectory / section?

Good suggestion. I'll implement it.

Reply:
New "backend" folder -> "using/backend/telemetry.md"